modules/pool/lib/pool-on-chain-data.service.ts

```diff
@@ -152,7 +158,14 @@ export class PoolOnChainDataService {
 
             if (isComposableStablePool(pool) || isWeightedPoolV2(pool)) {
                 // the new ComposableStablePool and WeightedPool mint bpts for protocol fees which are included in the getActualSupply call
-                multiPool.call(`${pool.id}.totalSupply`, pool.address, 'getActualSupply');
+                // getActualSupply fails for not initialized pools (pools where totalSupply = 0)
+                // until it is initialized, we'll use the totalSupply since it is also also updated but not 100% accurate it should be good enough
+                if (pool.dynamicData && pool.dynamicData?.totalSharesNum === 0) {
+                    multiPool.call(`${pool.id}.totalSupply`, pool.address, 'totalSupply');
```

Comment:
dunno if this is a good idea, just set it to 0

Reply:
scratch that, this will never get triggered then

Reply:
when you set it to 0, then it will never update and use the actualSupply